packages/cojson/src/coValueCore/coValueCore.ts

```diff
@@ -646,86 +645,235 @@ export class CoValueCore {
     return newContent;
   }
 
-  getValidTransactions(options?: {
-    ignorePrivateTransactions: boolean;
-    knownTransactions?: CoValueKnownState["sessions"];
-  }): DecryptedTransaction[] {
```

Comment:
Changed approach here, instead of using the knownState for the incremental processing I've moved to using sets to track the parsed transactions.

This way we can track both the parsed transactions from the branch and from the source.

Also refactored the `getValidTransactions` in two steps:
- transaction parsing, where the progressive computations is shared between all the content views
- `getValidTransactions` that returns the transactions based on the filters in input